pkg/label/handler.go

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2020. Ant Group. All rights reserved.
```

Comment:
We can update to `Copyright (C) 2022 Nydus Developers. All rights reserved.`. :)

Reply:
Done :)